keep/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "40ac95c0-35bd-49c8-a5f0-b21037bc87b4",
+  "app_id": "keep",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "An open-source AIOps Alert Management platform",
+    "title": "Keep",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Keep's providers page",
+        "image_url": "images/integrations-page.png"
+      },
```

Comment:
I'd recommend removing this image, as we'd like for this integration tile to focus on the connection with Datadog